src/30_Advanced/Embedding_videos_in_amp-accordion.html

```diff
@@ -0,0 +1,176 @@
+<!---{
```

Comment:
We should rename this sample to "Video Carousels with amp-carousel" corresponding to the already existing "Image Galleries with amp-carousel" sample.